### ISSUE TITLE
Task not display

### ISSUE THREAD

**Issue by xuankhanhdo** [heartexlabs/label-studio#913]
**Describe the bug**
Tasks are not displayed correctly

**To Reproduce**
1. Go to "Projects"
2. Click on a Task
3. Change to another task
4. See error : This second task are not display

**Expected behavior**
I have this problem when I use

- Option visible_when in a View

- Or when I use TextArea

      <View>
        <TextArea name="text_comments" toName="text" rows="3" placeholder="Name_label: value" editable="true" showSubmitButton="false"/>
      </View>

I used the same configuration in 1.0.1 and didn't have this problem
I'm using the version for local develpment, installed from master branch : 

https://github.com/heartexlabs/label-studio#install-for-local-development

**Screenshots**
![image](https://user-images.githubusercontent.com/15690936/117893731-41f2ed00-b2bb-11eb-8fbf-f4b1f28871db.png)

**Comment by visualcortex-team** [comment]
We're having a similar issue (tasks not displaying), but looking at the debug logs, we get this stack trace:
```
Traceback (most recent call last):
  File \"/usr/local/lib/python3.8/dist-packages/rest_framework/views.py\", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File \"/label-studio/label_studio/data_manager/api.py\", line 233, in get
    serializer = self.get_serializer_class()(task, many=False, context=context)
  File \"/label-studio/label_studio/data_manager/api.py\", line 211, in get_serializer_class
    return TaskSerializer
NameError: name 'TaskSerializer' is not defined
```
We're using the `:latest` docker image, updated 5 days ago.

**Comment by makseq** [comment]
@xuankhanhdo Please, try to update the master branch to the latest commit.

**Comment by makseq** [comment]
Solved in LS 1.1.
